README.md

```diff
@@ -1,128 +1,163 @@
+
+The web app serves three main purposes:
+
+- To promote and increase access to Japanese food.
+- To provide a service to the Japanese community in Australia and to those who love Japanese food products.
+- To contribute to business and sales growth for the client, Sakura Company.
+
+### Problem
+
+Prior to having the website, the primary means of obtaining Japanese food was through Japanese supermarkets, mainly located in central areas. This posed geographical constraints for those living in the suburbs. Moreover, limited store numbers and early closing times, often by 5pm, made it challenging for full-time workers to visit. Setting up a physical shop also presented a significant hurdle for Sakura Company, especially with rising rents due to recent unforeseen inflation.
+
+### Solution
+
+The launch of Sakura Pantry is envisioned to address these challenges. Sakura Pantry is an easy-to-use platform for purchasing Japanese food products, accessible to both city dwellers and those in the suburbs, operating 24/7. Unrestricted by the number of shops or closing times, it offers the convenience of online shopping, catering to the busy full-time workforce. By avoiding the need for physical shops, it provides a more cost-efficient means for Sakura Company to offer Japanese food products. In doing so, Sakura Pantry introduces a modern solution to access Japanese food ingredients in response to contemporary demands.
+
+## Functionality/Features
+
+The overall function of the website is to operate as an online store for selling Japanese food products. Below are the MVP features and functionality for the current stage of product development.
+
+### MVP Features
+
+- User authentication and authorisation (Register and Login)
+  - Allow users to create account and login.
+- Password recovery
+  - Allow users to reset password with reset link sent via email.
+- Site navigation and responsive design
+  - Intuitive and responsive UI/UX design for storefront to attract and engage audience.  
+- Search functionality
+  - Allow users to search products by name or by categories.
+- Product catalogue
+  - Showcase of new arrivals and featured products.
+  - Display a list of products when filtered by categories.
+- Product details
+  - Detailed information about each product, including product descriptions and product stock status.
+- Shopping cart
+  - Allow users to add products to cart, view items in cart and increase, decrease or remove items from cart.
+- Checkout process
+  - Allow users to fill out details of delivery and billing information.
+- Contact form
+  - Allow users to send their enquiries.
+- User account view
+  - View user profile
+  - Change password
+  - View order history
+- Admin dashboard:
+  - Summary view: allow admins to view total users, total products, total orders and total revenues.
+  - User management
+  - Product management
+  - Order management
+

```

Comment:
We also have Category management.

Reply:
I can add Category management here. But just clarify that in our wirefames, we dont have a separate tab for category management right? So how do admin CRUD categories, I dont think we went through that. 

Maybe we can include CRUD functionality for both products and categories in the Products tab? I will check and note this down in the wireframes too.

Reply:
I'll include the category management tab in the wireframe, my apologies for missing that earlier. We've already mentioned it in the sitemap, so it makes sense to have it here as well. Thank you.

Reply:
I'll include the category management tab in the wireframe, my apologies for missing that earlier. We've already mentioned it in the sitemap, so it makes sense to have it here as well. Thank you.

Reply:
I made the changes already. 

Can you review again?